### ISSUE TITLE
feat(trellis2-mv): sparse-structure flow + decode with multi-image conditioning

### ISSUE THREAD

**Issue by raven38** [raven38/pixal3d.cpp#62]
Parent: #58
Blocked by: #60, #61

## Objective

Wire the multi-image condition bank and generic MV sampler into the TRELLIS.2 **sparse-structure (SS) flow + decoder** stage.

This task ends at the decoded active voxel coordinate set. Do not implement shape/texture SLat here.

## Current single-image path

```text
cond512
 -> ss_flow.gguf (dense DiT, 12 FlowEuler steps)
 -> z_s [8,16,16,16]
 -> ss_dec.gguf
 -> logits [64^3]
 -> active coords @res32
```

The MV path must preserve the same noise, timestep, CFG, decoder and threshold semantics and only replace the positive-conditioning policy.

## Work

1. Add a stage function that accepts:
   - ordered `cond512[view]`
   - one negative condition
   - `MultiCondMode`
   - existing sampler params / seed-noise

2. Reuse the existing dense `DitRunner`; do not build one model graph per view.
   - upload a different condition tensor for each positive forward
   - preserve the same `x_t` for all per-view predictions at a multidiffusion step
   - negative forward remains one call

3. Decode the fused final latent through the existing SS decoder unchanged.

4. Add debug/fixture hooks sufficient to compare:
   - initial noise hash
   - final SS latent
   - active coord count + exact coord set
   - per-step selected view index in stochastic mode (debug only)

## Tests

- V=1 MV stage == current single-image stage.
- V=2/4 stochastic final latent vs #59 fixture.
- V=2/4 multidiffusion final latent vs #59 fixture.
- decoded coordinate set comparison against #59.
- swapping input order changes stochastic scheduling as expected.
- duplicate views are stable.
- no active coords remains a normal explicit failure.

## Acceptance

- [ ] SS MV stage passes #59 numerical/coord gates.
- [ ] Decoder code is unchanged except for test hooks if needed.
- [ ] Existing single-image SS tests remain green.
- [ ] Multidiffusion uses V positive calls and one negative call during guided steps.
- [ ] No extra full model copy per view.
- [ ] NaN/Inf checking remains active.

## Dependencies

- **Blocked by #60** — per-view cond bank.
- **Blocked by #61** — sampler fusion semantics.

## Complexity / agent budget

- **Complexity: 3/5** — localized stage integration with dense DiT parity.
- **Estimated agent budget: 100k–160k tokens.**
- Hard cap: **<300k**.

**Comment by raven38** [comment]
Delivered by #71 (squash-merged as d1b4926). On `main`: the multi-image condition bank and fusion sampler live in `src/flow_runner.cpp` / `include/flow_runner.h` (`MultiCondMode`), the CLI exposes `--trellis2-mv DIR` and `--trellis2-mv-mode stochastic|multidiffusion` (`src/trellis_args.cpp`), `src/trellis_cli.cpp:753-782` drives every cascade stage through that path, and the server serves `POST /generate-trellis2-mv` (`src/trellis-server.cpp:517`). Remaining work is tracked in #68 (test matrix) and #69 (confidence-weighted fusion).
